04pointer2/main.go

```diff
@@ -0,0 +1,33 @@
+package main 
+import "fmt"
+
+
+func main(){
+ type name struct{
+        
+       fname string
+       age int
+         }
+    
+    myguy:=name{
+        fname:"Nameless",
+        age: 25,
+    }
+
+
+// value before 
+
+fmt.Printf("FName Before :%s\n",myguy.fname)
+//value after
```

Comment:
I ll note that




Ayo Adesokan






On Thu, Jul 14, 2016 at 2:27 PM, Ewetumo Alexander <notifications@github.com
> wrote:

> In 04pointer2/main.go
> <https://github.com/adesokanayo/goclass/pull/2#discussion_r70804022>:
>
> > + type name struct{
> > +
> > +       fname string
> > +       age int
> > +         }
> > +
> > +    myguy:=name{
> > +        fname:"Nameless",
> > +        age: 25,
> > +    }
> > +
> > +
> > +// value before
> > +
> > +fmt.Printf("FName Before :%s\n",myguy.fname)
> > +//value after
>
> If you going to have comments, then let there be a space above the //value
> after
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adesokanayo/goclass/pull/2/files/3009a6727bef2ce69073bd87a5bd0f7e4f7bfb17#r70804022>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFINpkL_V2y7XgX_YVAx8poMeTb96D1Xks5qVjkzgaJpZM4JBmwX>
> .
>